financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentScreen.kt

```diff
@@ -327,6 +331,7 @@ internal fun ContentPreview(
     FinancialConnectionsPreview {
         ConsentContent(
             state = state.second,
+            topAppBarState = TopAppBarState(hideStripeLogo = state.second.consent()?.shouldShowMerchantLogos ?: true),
```

Comment:
unrelated - logos are always shown in v3 if I'm not mistaken, this is probably unneeded code now (don't change anything here, I'll create a task for it)

Reply:
Gotcha. Would be nice to get rid of the messy consent-specific code!